TODO.md

```diff
@@ -0,0 +1,17 @@
+# To-do list (temporary)
+
+All items here are subject for discussion.
+
+## Features
+
+### Filters
+
+* [ ] A way to filter by exact value of `level`? `level` + `level_exact`? or `level_ge` + `level_eq`?.
+
+### Other
+
+* [ ] `assert_one_pop()` to pop matching and assert only one.  Is it possible to provide custome error message with similar records?   Or full list if it has limitted size.
+* [ ] Add methods like `get_text()` to `LogPile` (note, it's also returned from `filter` and `drain`)?
+* [ ] Some way to automate `assert pile.is_empty()`? In `__exit__`?
```

Comment:
First I thought it doesn't have drawbacks. But now I remember a series of checks:
```
... run some code
... drain records
assert pile.is_empty()

... run more code
... drain records
assert pile.is_empty()
```
So when introducing auto-check we bring some inconsistency, since intermediate checks must be explicit. So it must be behind some option or marker at least.